wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/ParallelDeadlineGroup.java

```diff
@@ -74,7 +77,7 @@ public final void addCommands(Command... commands) {
     for (Command command : commands) {
       if (!Collections.disjoint(command.getRequirements(), m_requirements)) {
         throw new IllegalArgumentException(
-            "Multiple commands in a parallel group cannot" + "require the same subsystems");
+            "Multiple commands in a parallel group cannot require the same subsystems");
```

Comment:
```suggestion
            "Multiple commands in a parallel composition cannot require the same subsystems");
```
Does this fit in one line?

Reply:
Yes it does, though looking more at this, I'm not sure if changing "parallel group" to "parallel composition" makes sense (or at least, would fit within the scope of this PR, since there's lots of other references to parallel groups).